tests/components/recorder/test_init.py

```diff
@@ -16,6 +16,7 @@ class TestRecorder(unittest.TestCase):
 
     def setUp(self):  # pylint: disable=invalid-name
         """Setup things to be run when tests are started."""
+        recorder._INSTANCE = None
```

Comment:
SetUp happens before each test. If you want to have it before each file, you can use setUpModule